README.md

```diff
@@ -16,58 +16,121 @@ Open-source software-defined EDA
 
 </div>
 
-## Get running
+---
+
+## About
+
+### What \[is faebryk\]
+
+faebryk is an open-source software-defined EDA (electronic design automation) tool.
+Think of it like the next step from EDA tools like KiCAD, Altium, Eagle...
+in the way those were the next step from designing electronic circuits on paper.
+While the main focus is on the EDA part currently, faebryk aims to become a wholistic system design tool.
+The main concept of faebryk consists of **describing your system** instead of describing its implementation, and iteratively providing implementation details as needed.
+
+### How \[does faebryk work\]
+
+faebryk itself is just a **python library** that you include in your project. It is providing you with all the tools to describe and design your system and to export that design into something useful like for example a netlist, a devicetree, gerbers etc, which you then can use in the next steps of your project. Key concepts of faebryk are the graph, importers, exporters, the library and the user application.
+To understand how to use faebryk in your project see #Using faebryk.
+
+### Who \[is faebryk\]
+
+faebryk is a community driven project. That means faebryk does not only consist out of core developers but also users, external contributors, moderators and you! It is founded by a group of embedded, electrical, product design and software engineers with a love for making.
+
+### Why \[do we make faebryk\]
+
+The founders of the project noticed that the innovations of software engineering that make fast, scalable, robust solutions possible have not found their way into hardware design. Furthermore there is a lot of duplicate work done. Think of determining the pinout of a SoC and then having to translate that exact pinout into software again, or having to constantly adapt designs for supply chain issues.
+Next to that they have assessed that hardware design has quite a big barrier of entry for makers which is not necessary or desirable.
+Currently hardware design is also quite manual labor intensive with very little automation.
+faebryk aims to tackle all those issues and more like proper version management and collaboration.
+
+### When \[is faebryk being developed\]
+
+faebryk is being continuously developed.
+The core team focuses on the core functionality and features of faebryk, and general direction of the project.
+The strength of the community can really shine with the development of importers, exporters, libraries, and projects, but everyone is welcome to help out where they can.
+faebryk uses semantic versioning in the releases.
```

Comment:
This does not fit in this section. If we add something about stability it would go there.